release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -268,6 +268,41 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.8 includes the following alerting rule changes:
+
+.Alerting rule changes
+[%collapsible]
+====
+* The `ThanosSidecarBucketOperationsFailed` alert is added. The warning alert provides notification if the Thanos sidecar bucket operations are failing.
+* The `ThanosSidecarPrometheusDown` alert severity is updated from _critical_ to _warning_.
+* The `ThanosSidecarUnhealthy` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryRangeErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryInstantLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for instant queries.
+* The `ThanosQueryRangeLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for range queries.
+* For all Thanos Querier alerts, the `for` duration is increased to 1 hour.
+* For all Thanos sidecar alerts, the `for` duration is increased to 1 hour.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+=====
+====
+
+[id="ocp-4-8-monitoring-kube-state-metrics-2.0.0-upgrade"]
+==== kube-state-metrics upgraded to version 2.0.0
+
+`kube-state-metrics` is upgraded to version 2.0.0. For more information on all changes in this version, including updates to metrics and flags, see the link:https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md[kube-state-metrics CHANGELOG].
```

Comment:
@vikram-redhat @sferich888 Am I okay to link out to the `kube-state-metrics` change log? The [`cluster-monitoring-operator` 4.8 CHANGELOG](https://github.com/openshift/cluster-monitoring-operator/blob/master/CHANGELOG.md#48) has this entry:

> #1093 Bump kube-state-metrics to major new release v2.0.0-rc.1. This changes a lot of metrics and flags, see kube-state-metrics CHANGELOG for full changes.

Reply:
Updated v2.0.0 to v2.0.0-rc.1, but same questions applies as far as linking to the `kube-state-metrics` change log. ^

Reply:
`#pod_name=$(oc -n openshift-monitoring get pod | grep kube-state-metrics | awk '{print $1}'); echo $pod_name;  oc -n openshift-monitoring exec -c kube-state-metrics $pod_name -- kube-state-metrics --version
kube-state-metrics-5d7c8d649f-fvt9k
version.Version{GitCommit:"7444dc4", BuildDate:"2021-06-02T15:39:06Z", Release:"v2.0.0", GoVersion:"go1.16.1", Compiler:"gc", Platform:"linux/amd64"}
`
Got version is v2.0.0, not sure why change from v2.0.0 to v2.0.0-rc.1

Reply:
The worry by including the link is if all those changes are in the downstream and supported. Do need to get @sferich888 to ack this first.

Reply:
We should not be linkint to the github change log; we should be including / incorporating it into our documentation.